photon-server/src/main/java/org/photonvision/common/hardware/PWM/CustomPWM.java

```diff
@@ -18,10 +18,11 @@
 package org.photonvision.common.hardware.PWM;
 
 import org.photonvision.common.configuration.HardwareConfig;
+import org.photonvision.common.hardware.Platform;
 
 public class CustomPWM extends PWMBase {
     private int pwmRate = 0;
-    private int[] pwmRange = new int[2];
+    private int pwmRange = 0;
```

Comment:
Why did this change from an array? It's possible to want a lower limit to the range